packages/api-v2/src/hooks/useGrowthbookGetFeatureValue.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { useIsMounted } from 'usehooks-ts';
+
+import { getFeatureFlag } from '@deriv/utils';
+import { Analytics } from '@deriv-com/analytics';
+
+interface UseGrowthbookGetFeatureValueArgs<T> {
+    featureFlag: string;
+    defaultValue?: T;
+}
+
+const useGrowthbookGetFeatureValue = <T extends string | boolean>({
+    featureFlag,
+    defaultValue,
+}: UseGrowthbookGetFeatureValueArgs<T>) => {
```

Comment:
We already have a `useGrowthbookIsOn` hook in api-v2. Can't that hook be used and extended (if needed) instead?

Reply:
useGrowthbookIsOn is only returning boolean values. 🙏 I'm trying to get JSON values instead from `Growthbook` that's why i can't use the hooks

Reply:
we have useGrowthbookGetFeatureValue in @deriv/hooks for this purpose. isnt?

Reply:
Yes but wallets are not using @deriv/hooks 🙏